app/Phragile/Burndown.php

```diff
@@ -0,0 +1,107 @@
+<?php
+namespace Phragile;
+
+class Burndown {
+	// TODO: ideally these should come from a cached call to maniphest.querystatuses
+	private static $STATUS_OPEN = ['stalled', 'open'];
+
+	public function __construct(\Sprint $sprint, TaskList $tasks, PhabricatorAPI $phabricator)
+	{
+		$this->sprint = $sprint;
+		$this->tasks = $tasks;
+		$this->phabricator = $phabricator;
+		$this->days = $this->daysInSprint();
+	}
```

Comment:
You are doing work in the constructor by computing the `days` field. Not necessary bad, though definitely a code smell. Looks like the field is only accessed from one of the methods. Do you expect that method to be called multiple times? If not, then there is no need for this. If yes, then you can lazily compute the thing when needed.